agent/agents/mysql/slowlog/parser/parser.go

```diff
@@ -167,20 +168,21 @@ func (p *SlowLogParser) Run() {
 func (p *SlowLogParser) parseHeader(line string) {
 	p.logf("header")
 
-	if !headerRe.MatchString(line) {
+	if !isHeader(line) {
 		p.inHeader = false
 		p.inQuery = true
 		p.parseQuery(line)
 		return
 	}
+	line = line[2:] // without header prefix and space
```

Comment:
you can move `line[2:]` to a self explaining function, this will allow you to get rid of comment

Reply:
done